docs/CONTRIBUTING.md

```diff
@@ -54,7 +54,7 @@ Once you clone the repository, you will have a monorepo which consists of a bunc
 - `handler-*` are utility packages to create serverless function handlers
 - `serverless-*` are packages containing infrastructure components used to deploy your infrastructure
 
-`sample-project` folder is the place that simulates a project structure of a project created using `create-webiny-project`. This is your development sandbox.
+`api` and `apps` folders in root of project directory are there to simulates a project created using `create-webiny-project`. This is your development sandbox.
```

Comment:
I'd rephrase this completely:

In the root of the project, you'll find the `api` and `apps` folders. These are the [stacks](https://docs.webiny.com/docs/deep-dive/stacks) we use as our sandbox for development of Webiny packages. The setup of the repo is identical to the one created by `create-webiny-project`.